sphinx/Makefile

```diff
@@ -60,7 +60,7 @@ slides:
 	@echo "Build finished. The slide-html files are in $(BUILDDIR)/slides."
```

Comment:
Sphinx stuff need real thought...

Reply:
I wouldn't worry about the docs for now.